huaweicloud/services/cce/resource_huaweicloud_cce_addon.go

```diff
@@ -379,23 +370,22 @@ func waitForAddonDelete(cceClient *golangsdk.ServiceClient, id, clusterID string
 		if r.Status.Status == "Deleting" {
 			return r, "Deleting", nil
 		}
-		log.Printf("[DEBUG] CCE Addon %s still available.\n", id)
+		log.Printf("[DEBUG] CCE add-on %s still available.", id)
```

Comment:
```suggestion
		log.Printf("[DEBUG] CCE add-on %s is still available.", id)
```

Reply:
fixed and bring this file change to another PR.